src/java/org/apache/cassandra/tcm/CMSOperationsMBean.java

```diff
@@ -33,6 +35,7 @@ public interface CMSOperationsMBean
     public void cancelReconfigureCms();
 
     public Map<String, String> describeCMS();
+    public boolean isCurrentMember(InetAddressAndPort peer);
```

Comment:
we can't have InetAddressAndPort here since we expose this over JMX - we need to translate between strings